### DISCUSSION TITLE
Rooms / Points / Zones?

### DISCUSSION THREAD

**Discussion by PilgrimToHyperion** [nergal/homeassistant-vacuum-viomi#24] [Q&A]
I was very glad to find this and how well it works to detect my viomi.v8 model. Thank you for that first of all. I'm new to Home Assistant (day 3!) and couldn't figure out whether this supports using rooms/ points / zones as is possible from this card - [https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card](url)

I know this is early alpha but I'm so super glad you're working on it.

**Comment by Tontze** [comment]
This is what i was wondering too, couldnt get predefined zones to work

**Reply by yayaya142** [reply]
Did you find any solution for this?
